inst/conf/yarn-config-template.yml

```diff
@@ -0,0 +1,9 @@
+default:
```

Comment:
If we are going to make these a default for `master = "yarn-*"` are these values truly universally desirable? (i.e. do they depend significantly on hardware or can they be taken for granted as a reasonable baseline for all clusters).